backend/app/views/spree/admin/products/_form.html.erb

```diff
@@ -172,6 +172,24 @@
           </div>
         </div>
       <% else %>
+        <div data-hook="admin_product_form_gtin">
+          <%= f.field_container :gtin do %>
```

Comment:
Can we be explicit here and tell users that this is the "Primary" or "Product GTIN" to better distinguish it from the variant GTINs? Since this is solely used if the product has no variants. 

See above that we call the SKU "Master SKU" (for the lack of a better name. "Master" is obviously a bad naming. I would prefer "Main" or "Primary", but am open for suggestions)

Reply:
Yeah, go crazy, @rahulsingh321 do that!

Reply:
> See above that we call the SKU "Master SKU" (for the lack of a better name. "Master" is obviously a bad naming. I would prefer "Main" or "Primary", but am open for suggestions)

I thought about the whole main / primary / master box on product and variant.

I don't know about Shopify, but I can tell you from BC it's not separately labeled.

The context of the page is very strong as you have to literally go from product list over product page to variant to varian detail page. 

I appreciate the need for clarity, but I do not think anything is unclear if we don't have the same resource or field twice in the same field. 

In blows up the language translation and does not serve, in my opinion, any purpose!

Reply:
@rahulsingh321 please go with Master on Product Pages till a general solution has been found to the naming issue. 

@tvdeyen @kennyadsl can you live with that? We do not want to blow up this PR just to add / remove labels from 5-6 fields.

Reply:
Yes,  this was my intention. Thanks